MaskRCNN/config.py

```diff
@@ -139,6 +139,8 @@ def __ne__(self, _):
 # Un-scaled version: [0.485, 0.456, 0.406], [0.229, 0.224, 0.225]
 _C.PREPROC.PIXEL_MEAN = [123.675, 116.28, 103.53]
 _C.PREPROC.PIXEL_STD = [58.395, 57.12, 57.375]
+_C.PREPROC.PREDEFINED_PADDING = True
+_C.PREPROC.PADDING_SHAPES = [(800, 1000), (800, 1100), (800, 1200), (800, 1300), (800, 1350)]
```

Comment:
Add a comment here that while you should only pass in the landscape shapes, we will still create tensors with portrait shape

Reply:
Added. Also noted that these should be listed in decreasing h/w order. We can remove this requirement when we revise the algorithm.